thoth/storages/graph/models.py

```diff
@@ -85,9 +85,31 @@ class RuntimeEnvironment(VertexBase):
 
 
 class SoftwareStack(VertexBase):
-    """Observations we have about the given stack."""
+    """A software stack crated by packages in specific versions."""
 
-    # TODO: add observation info and info about origin
+    installable = VertexProperty(properties.Boolean, default=True)
+
+
+class SoftwareStackObservation(VertexBase):
+    """Observations we have about the given stack based on run on a specific hardware."""
+
+    performance_index = VertexProperty(properties.Float)
+    observations_document_id = VertexProperty(properties.String)
+
+
+class HardwareInformation(VertexBase):
+    """Hardware specification and propertires."""
+
+    cpu_vendor = VertexProperty(properties.String)
+    cpu_model_name = VertexProperty(properties.String)
+    cpu_cores = VertexProperty(properties.Integer)
```

Comment:
we need cpu_model:unsighed_int and cpu_family:unsigned_int too :)

@sub-mod can you give some advice what is required for TF build/run?